data/hai/hai ships.txt

```diff
@@ -1736,7 +1736,7 @@ ship "Violin Spider"
 		"drag" 0.9
 		"heat dissipation" .7
 		"outfit space" 92
-		"weapon capacity" 19
+		"weapon capacity" 22
```

Comment:
```suggestion
		"weapon capacity" 23
```

Going to go ahead and suggest this directly, here, so there's a specific spot to discuss this change versus the original suggestion. So probably don't commit it until that discussion takes place. :)

I also think it's a positive one (especially with the reminder that is was the initial combo when integrated). This also opens them up to 3 functional loadouts even just with Hai tech, which helps justify it's sole place in the role.

We don't need to add all those variants immediately in this PR, though.

Reply:
Yeah I was thinking of keeping the loadouts for another PR once we're sure what weap space to go with
I'll apply this since QuantumShark agrees with it, but I'll also keep the discussion open for if smbd disagrees